platform/admin/core/admin-core-appservice/pom.xml

```diff
@@ -108,17 +124,21 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
                             platform-util,
-                            ddf-security-common
+                            ddf-security-common,
+                            boon
                         </Embed-Dependency>
                         <Import-Package>
                             !org.codice.ddf.platform.util.http,
+                            !com.sun.management,
```

Comment:
Honestly, I'm not sure why this was in here since I don't remember adding it. Also it's actually ensuring that a specific package was not imported. @brjeter any ideas?